src/app/portlets/dot-edit-page/dot-edit-page-resolver.service.ts

```diff
@@ -15,6 +15,8 @@ export class PageViewResolver implements Resolve<any> {
      * @memberof PageViewResolver
      */
     resolve(route: ActivatedRouteSnapshot): Observable<DotPageView> {
-        return this.pageViewService.get(route.queryParams.url);
+        return this.pageViewService.get(
+            route.queryParams.url !== undefined ? route.queryParams.url.replace(/^\//, '') : null
```

Comment:
Do this in the get of the pageViewService, for me it make more sense there since there is where we do the request, the resolver shouldn't care about this.